frontend/src/modules/Shares/components/AddShareItemModal.js

```diff
@@ -20,6 +20,7 @@ import { Defaults, Pager, Scrollbar } from 'design';
 import { SET_ERROR, useDispatch } from 'globalErrors';
 import { useClient } from 'services';
 import { addSharedItem, getShareObject } from '../services';
+import { generateShareItemLabel } from '../../../utils/share';
```

Comment:
Since v2.0 we avoid relative imports in the frontend and replace them with global imports. In case the share.js is places inside the utils, it needs to be added to the `index.js` of the utils and imported directly from `../utils`

Reply:
I added it to index.js. 
I will still need to import it like this:
`import { generateShareItemLabel } from '../../../utils';`
Does that look okay?